README.md

```diff
@@ -19,15 +19,15 @@
   - [Examples](#examples)
     - [Built-in Sanity auth](#built-in-sanity-auth)
       - [Using the `/pages` directory](#using-the-pages-directory)
-      - [Using the `/app` directory (experimental)](#using-the-app-directory-experimental)
+      - [Using the `/app` directory](#using-the-app-directory-experimental)
     - [Custom token auth](#custom-token-auth)
       - [Using the `/pages` directory](#using-the-pages-directory-1)
-      - [Using the `/app` directory (experimental)](#using-the-app-directory-experimental-1)
+      - [Using the `/app` directory](#using-the-app-directory-experimental-1)
```

Comment:
Shouldn't the `#` links also need to be updated?

Reply:
Thanks for mentioning I fixed it here: #432